cmd/podman/build.go

```diff
@@ -252,3 +270,13 @@ func buildCmd(c *cli.Context) error {
 
 	return runtime.Build(getContext(), options, dockerfiles...)
 }
+
+// Tail returns a string slice after the first element unless there are
+// not enough elements, then it returns an empty slice.  This is to replace
+// the urfavecli Tail method for args
+func Tail(a []string) []string {
+	if len(a) >= 2 {
+		return []string(a)[1:]
+	}
+	return []string{}
+}
```

Comment:
Is there a common area we could/should tuck this away into like you did in Buildah?

Reply:
sure, but build is the only one to use it.  this will get reworked completely with the remote client build which i am working on in parallel. deferring to that pr